pkg/csi/service/node.go

```diff
@@ -631,9 +632,32 @@ func (s *service) NodeGetInfo(
 	if nodeID == "" {
 		return nil, status.Error(codes.Internal, "ENV NODE_NAME is not set")
 	}
+	var maxVolumesPerNode int64
+	if v := os.Getenv("MAX_VOLUMES_PER_NODE"); v != "" {
+		if value, err := strconv.ParseInt(v, 10, 64); err == nil {
```

Comment:
Should we default to `maxAllowedBlockVolumesPerNode` if the value is < 0 or > max? Instead of failing here

Reply:
Or default to 0?

Reply:
I think when the configuration is wrong we should fail, so the admin knows about the right parameter value they need to set. if we default to 0 or max 59, the admin may assume, the configuration he provided is correct.